src/utils/common.js

```diff
@@ -1,5 +1,19 @@
 const WorkflowError = require("./errors/workflow_error");
 const URL = require("url").URL;
+const yaml2json = require('js-yaml');
+const fs = require('fs/promises');
+const path = require('path');
+
+const schema = [];
+console.log("the module is being run bruhv");
+
+exports.getSchema = async () => {
+  console.log(schema);
```

Comment:
Useful logging during development, however once the module is ready for merging these should be removed (or replaced with more concise `debug` calls